lib/board.rb

```diff
@@ -1,7 +1,12 @@
 class Board
-  def initialize
-    @board = create_board
+  def initialize(cells = [], last_move = 'o')
+    if cells.empty?
+      @board = create_board
+    else
+      @board = cells
+    end
     @turn = 'x'
+    @last_move = last_move
```

Comment:
not sure what 'last_move' indicates having come across it for the first time.. Hopefully all will be revealed, but initially I can't think what it means